WcaOnRails/app/models/post.rb

```diff
@@ -29,25 +29,25 @@ def body_teaser
   CRASH_COURSE_POST_SLUG = "delegate-crash-course"
 
   def self.crash_course_post
-    post = ( Post.find_by_slug(CRASH_COURSE_POST_SLUG) ||
-             Post.create!(slug: CRASH_COURSE_POST_SLUG, title: "Delegate crash course", body: "Nothing here yet") )
+    post = (Post.find_by_slug(CRASH_COURSE_POST_SLUG) ||
+            Post.create!(slug: CRASH_COURSE_POST_SLUG, title: "Delegate crash course", body: "Nothing here yet"))
```

Comment:
Neat! I've created https://github.com/thewca/worldcubeassociation.org/issues/1299 and assigned it to myself. I'll take care of that in a seperate PR after this one.